code/game/jobs/squads.dm

```diff
@@ -148,8 +149,9 @@ GLOBAL_LIST_EMPTY(helmetmarkings_sl)
 
 	if(istype(H.wear_ear, /obj/item/radio/headset/almayer)) // they've been transferred
 		var/obj/item/radio/headset/almayer/headset = H.wear_ear
-		if(headset.sl_direction)
-			SSdirection.start_tracking(tracking_id, H)
+		if(headset.sl_direction && H.mind.assigned_role != "Squad Leader")
```

Comment:
Better use something like `if(H != H.squad.squad_lead)` instead of `H.mind.assigned_role != "Squad Leader"`.